woocommerce-gateway-ebanx/includes/class-wc-ebanx-baloto-gateway.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+if (!defined('ABSPATH')) {
+	exit;
+}
+
+class WC_EBANX_Baloto_Gateway extends WC_EBANX_Gateway
+{
+	/**
+	 * Constructor
+	 */
+	public function __construct()
+	{
+		$this->id           = 'ebanx-baloto';
+		$this->method_title = __('EBANX - Baloto', 'woocommerce-gateway-ebanx');
+
+		$this->api_name    = 'baloto';
+		$this->title       = __('Baloto', 'woocommerce-gateway-ebanx');
+		$this->description = __('Paga con boleta Baloto.', 'woocommerce-gateway-ebanx');
```

Comment:
Please, talk to Larissa or Liciane if this text is right. Maybe just `Paga con Baloto` resolves.